### PR TITLE
Add instructions for creating a new CNCF slack account

### PR DESCRIPTION
**Pull request by tedsuo**
http://slack.cncf.io/ <-- new users need to go here first to create an account.

### CODE REVIEW

**Review by yurishkuro — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ Both committes meet regularly and the respective meeting notes are publicly avai
 ### Discussions
 We use [GitHub discussions](https://github.com/open-telemetry/community/discussions) for most communications. Please join us there!
 
-For those who are brand new to OpenTelemetry and just want to chat or get redirected to the appropriate place for a specific question, feel free to join [the CNCF OpenTelemetry Slack channel](https://cloud-native.slack.com/archives/CJFCJHG4Q).
+For those who are brand new to OpenTelemetry and just want to chat or get redirected to the appropriate place for a specific question, feel free to join [the CNCF OpenTelemetry Slack channel](https://cloud-native.slack.com/archives/CJFCJHG4Q). If you are new, you can create a CNCF Slack account [hare](http://slack.cncf.io/).
```
Comment (yurishkuro) [inline README.md:41]:
```suggestion
For those who are brand new to OpenTelemetry and just want to chat or get redirected to the appropriate place for a specific question, feel free to join [the CNCF OpenTelemetry Slack channel](https://cloud-native.slack.com/archives/CJFCJHG4Q). If you are new, you can create a CNCF Slack account [here](http://slack.cncf.io/).
```

Comment (bhs) [inline README.md:41]:
```suggestion
For those who are brand new to OpenTelemetry and just want to chat or get redirected to the appropriate place for a specific question, feel free to join [the CNCF OpenTelemetry Slack channel](https://cloud-native.slack.com/archives/CJFCJHG4Q). If you are new to the CNCF Slack workspace, you can create an account [here](http://slack.cncf.io/).
```

Comment (bhs) [reply README.md:41]: @tedsuo take your pick of these two suggestions :)

**Review by bhs — APPROVED** [review]
Approved after fixing the typo in either of the ways suggested here :)

File: README.md (hunk 1 above)
Comment (tedsuo) [reply README.md]: saw the other one first, Yuri beat you by a hare

**Comment by tedsuo** [comment]
@bhs updated!

**Review by SergeyKanzhelev — APPROVED** [review]

**Pull request merged**